packages/web/src/components/pages/Webhooks/WebhookDetails/ConditionTab/Condition.tsx

```diff
@@ -68,6 +70,14 @@ const Condition: FC = () => {
       initialValues={{events: initialEvents, labels: initialLabels}}
       disabled={!mayEdit}
       onConfirm={onFinish}
+      footer={
+        <>
+          Learn more about{' '}
+          <ExternalLink href={externalLinks.notificationsAndWebhooks} target="_blank">
```

Comment:
Could you point to Webhook Condition section?

https://docs.testkube.io/articles/webhooks/#resource-selector-labels